lib/src/runner/configuration.dart

```diff
@@ -124,6 +138,10 @@ class Configuration {
   Glob get filename => _filename ?? defaultFilename;
   final Glob _filename;
 
+  /// The maximum length to print per line.
+  int get lineLength => _lineLength ?? _defaultLineLength;
```

Comment:
Add a TODO to make this global again when dart-lang/sdk#24716 is fixed.

Reply:
done